lib/lighthouse/audit_service.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'json'
+
+# Compares a url's actual score to the expected score.
+class AuditService
+  def initialize(url, audit, score)
+    @url = url
+    @audit = audit
+    @score = score
+    @port = Lighthouse::Matchers.remote_debugging_port
+    @runner = Lighthouse::Matchers.runner
+    @cmd = Lighthouse::Matchers.lighthouse_cli
+  end
+
+  def passing_score?
+    actual_score >= @score
+  end
+
+  private
+
+  def opts
+    "'#{@url}' --quiet --output=json #{"--port=#{@port}" if @port}".strip
+  end
+
+  def output
+    @runner.call("#{@cmd} #{opts}")
+  end
+
+  def results
+    JSON.parse(output)
+  end
+
+  def actual_score
```

Comment:
`measured_score`? :elephant: - `actual` bothers me for some reason

Reply:
Cool. This was the original language but happy to change it.

Reply:
:+1: